src/dev/src/app/searchfield/searchfield.demo.html

```diff
@@ -6,7 +6,7 @@
 <h2>Search Field</h2>
 
 <div class="clr-example">
-  <div class="main-container">
+  <div class="main-container clr-col-md-4">
```

Comment:
in this demo we don't care about the styling.
but if you want the input to be smaller set it directly to it